src/shared/ui/select/MultipleSelectList.tsx

```diff
@@ -0,0 +1,77 @@
+import { useRef, useState } from 'react';
+import ArrowDownIcon from '@/shared/assets/arrow-down.svg?react';
+import ArrowUpIcon from '@/shared/assets/arrow-up.svg?react';
+import { useOutsideClick } from '@/shared/hooks/useOutsideClick';
+import { RemoveButton, AddButton } from '@/shared/ui';
+import * as S from '@/shared/ui/select/MultipleSelectList.style';
+export interface SelectItem {
+  id: number;
+  item: string;
+}
+export interface MultipleSelectListProps {
+  items: SelectItem[];
+  selectedItems: { id: number; item: string }[];
+  onSelect: React.Dispatch<React.SetStateAction<SelectItem[]>>;
+  selectLimit: number;
+  placeholder: string;
+}
+export const MultipleSelectList = ({
+  items,
+  selectedItems,
+  onSelect,
```

Comment:
이렇게가 실제로는 하나의 쌍인데  이름으로 드러나지가 않아서 표현해주면 좋을 것 같습니다

대략 이런 느낌 혹은
```suggestion
  selectedItems,
  onClickSelectedItems,
```

요런 느낌
```suggestion
  selectedItems,
  addSelctedItem,
  deleteSelctedItem
```

Reply:
props로 한 쌍으로 받을때는 인지가 되도록 네이밍 해야되겠네요 감사합니다!
https://github.com/GrebbMe/grabbme-fe/pull/54/commits/4e327b2d379a92355c1197256366950fd2b74e9d